_posts/2017-02-07-women-technology-careers.md

```diff
@@ -0,0 +1,22 @@
+
+Regarding schools, the only presentation I was subjected to regarding coding was a robot and that was after I had applied to study Software Engineering at Sheffield Hallam University, you can imagine my thrill of excitement. Where is the drive to even pull girl’s interests into the likes of technology? Every campaign I have seen has been male orientated, thus increasing the stigma that it’s a male aimed presentation, and a male labelled role. Why not produce talks on web design and application building that involves such stereo-typing girl hobbies as street dancing, fashion or beauty? Why is this not happening? It’s very hypocritical for me to sit her and pledge to you about the hypocrisy of wanting girls to get into technology by using stereo-types, however this is the only way to engage interest and actually allow social acceptance. If coding is seen as a female and male gender acceptance role, then if either sex holds an interest in it, they can be allowed to break the chains without a concern of a social back lash. How great would that be?  This would stem the stigma and build the interest for women in the technology sector rapidly, by simply starting at the bottom. Bring technology further into the schools, let children see both men and women presenting in all sectors, allow children’s minds to grow and develop with knowledge and confidence. This is the only way (I believe) to change this situation and actually increase numbers of females rapidly into the sector. However, what do I know, I’m down as the 2% of female geeks that simply lives and breathes code with interest.
```

Comment:
"girls'", not "girl's".

"male-oriented", not "male orientated".
"male-aimed", not "male aimed".
"male-labelled", not "male labelled".

"stereotypical", not "stereo-typing".

Place quote marks around "girl hobbies" to show that it is a perception rather than a literal "these are only for girls".

"sit here", not "sit her".

I don't think you mean "pledge": how about just "talk"?

No hyphen in "stereotypes".
No space in "backlash".